intellij-setup.md

```diff
@@ -5,7 +5,7 @@ sidebar: home_sidebar
 permalink: /intellij-setup/
 ---
 
-This page describes how to set up IntelliJ IDE to develop Flutter apps.
+This page describes how to set up an IntelliJ IDE to develop Flutter apps.
```

Comment:
s/an/the?

(Having said which this is your area of expertise not mine, so I'll happily stand corrected :))